### PR TITLE
Add Traditional Chinese Translation

### PR DESCRIPTION
**Pull request by xMikux**
And rename zh to zh_cn.

### CODE REVIEW

File: bans-core/src/main/resources/lang/messages_zh_tw.yml [hunk 1]
```diff
@@ -0,0 +1,366 @@
+#Messages configuration
+#In most cases, the variables inside the default messages are those available
+#in that specific message. The exception to this is messages which are related
+#to a certain punishment.
+#When message has an associated punishment, multiple variables are available:
+#%ID% - punishment ID number
```
Comment (A248) [inline bans-core/src/main/resources/lang/messages_zh_tw.yml:6]: Why were the spaces before and after the comment character removed? It used to look like this:
```
 # When message has an associated punishment, multiple variables are available:
 # %ID% - ...
```

Now it looks like this:
```
#When message has an associated punishment, multiple variables are available:
#%ID%
```
Note that the space before the `#` and the space after the `#` are missing.

Comment (xMikux) [reply bans-core/src/main/resources/lang/messages_zh_tw.yml:6]: hmm, maybe because i using crowdin to translate, so it auto remove, i will try to add back

Comment (xMikux) [reply bans-core/src/main/resources/lang/messages_zh_tw.yml]: OK, I add all back

**Review by KoxSosen — APPROVED** [review]
Looks good, LGTM. Thank you for contributing.

**Pull request merged**